cou/utils/juju_utils.py

```diff
@@ -260,8 +271,12 @@ async def async_get_application_config(application_name, model_name=None):
 
 
 async def async_run_action(
-    unit_name, action_name, model_name=None, action_params=None, raise_on_failure=False
-):
+    unit_name: str,
+    action_name: str,
+    model_name: Optional[str] = None,
+    action_params: Optional[Dict] = None,
+    raise_on_failure: bool = False,
+) -> Action:
     """Run action on given unit.
 
     :param unit_name: Name of unit to run action on
```

Comment:
change :type model_name: str to :type model_name: Optional[str]
change :type action_params: dict to :type action_params: Optional[Dict]
change :param raise_on_failure: Raise ActionFailed exception on failure to  :param raise_on_failure: Raise ActionFailed exception on failure, defaults to False

Reply:
ok